Description: Upstream changes introduced in version 0.1-0.0.1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ros-electric-interaction-quickdev-0 (0.1-0.0.1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
 .
 The person named in the Author field signed this changelog entry.
Author: Edward T. Kaszubski <ekaszubski@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ros-electric-interaction-quickdev-0-0.1.orig/cpp0x/Makefile
+++ ros-electric-interaction-quickdev-0-0.1/cpp0x/Makefile
@@ -1 +1 @@
-include $(shell rospack find mk)/cmake.mk
\ No newline at end of file
+include $(shell rospack find quickdev_build)/cmake.mk
--- ros-electric-interaction-quickdev-0-0.1.orig/cpp11/Makefile
+++ ros-electric-interaction-quickdev-0-0.1/cpp11/Makefile
@@ -1 +1 @@
-include $(shell rospack find mk)/cmake.mk
+include $(shell rospack find quickdev_build)/cmake.mk
--- ros-electric-interaction-quickdev-0-0.1.orig/quickdev_build/cmake.mk
+++ ros-electric-interaction-quickdev-0-0.1/quickdev_build/cmake.mk
@@ -29,8 +29,14 @@ force_remake:
 	@make clean && make
 	@echo "-- << Done rebuilding project"
 
-distclean:
-	@make clean
+debclean:
+	-rm -rf build
+
+distclean: clean
+	@echo "Removing any binary files"
+	@-rm -rf build
+	@-rm -rf lib
+	@-rm -rf bin
 
 test: all
 	@if [ ! -r build ]; then \
@@ -55,10 +61,6 @@ PACKAGE_NAME=$(shell basename $(PWD))
 clean:
 	@echo "-- >> Cleaning project..."
 	@-if [ -r build ]; then cd build && make clean; fi
-	@echo "Removing any binary files"
-	@-rm -rf build
-	@-rm -rf lib
-	@-rm -rf bin
 	@echo "Removing any auto-generated docs"
 	@-rm -rf docs
 	@echo "Removing any auto-generated messages"
--- ros-electric-interaction-quickdev-0-0.1.orig/quickdev_build/cmake_stack.mk
+++ ros-electric-interaction-quickdev-0-0.1/quickdev_build/cmake_stack.mk
@@ -3,7 +3,7 @@ CMAKE_FLAGS= -Wdev -DCMAKE_TOOLCHAIN_FIL
 
 # make sure we default to all
 all:
-	rosmake
+	rosmake --no-rosdep
 
 remake:
 	make clean && make
